### ISSUE TITLE
Accessibility: Cannot tab to uncheck Removed/Added

### ISSUE THREAD

**Issue by j-der** [twg/wikiwash#209]
**Issue:** Using keyboard-only navigation, users cannot use tab key to check/uncheck Removed and Added checkboxes
**Browser:** Chrome 50.0.2661.102
**OS:** OSX El Capitan 10.11.5

**Steps to reproduce:**

1. Search for "Democracy" from WikiWash main page (or use any search term)
2. Press the 'tab' key on your computer. WikiWash logo will be focused.
3. Press 'tab' again, About button is focused.
4. Press 'tab' again, Search bar is focused.
5. Press 'tab' again, focus moves to link for 'Download as CSV'

In Chrome Developer Tools, removing `display: none` from `input[type=checkbox], input[type=radio]` in CSS reveals standard checkboxes. In this case, the user is able to tab to check or uncheck. See screenshot attached.
<img width="1248" alt="screen shot 2016-05-29 at 9 36 31 pm" src="https://cloud.githubusercontent.com/assets/13560752/15637672/ac935026-25e6-11e6-8a04-7cadb97b8a0e.png">
